test/capture.js

```diff
@@ -1,3 +1,21 @@
+define([ 'intern!tdd',
```

Comment:
This indentation is kinda whacky. I'd suggest:

```js
define([
    'intern!tdd',
    '...'
], function(tdd, expect) {
// no indentation here, otherwise the whole file would be indented
});
```